modules/autofill/autofill-impl/src/main/res/values/strings.xml

```diff
@@ -15,6 +15,7 @@
   -->
 
 <resources>
+    <string name="management_screen_title">Autofill Logins</string>
     <string name="management_locked_mode_text">Unlock Autofill</string>
     <string name="management_disabled_mode_cta"><b>Set up in Settings</b></string>
     <string name="management_disabled_mode_subtitle">A device lock pattern, PIN, or biometrics are required to protect your Autofill Login details.</string>
```

Comment:
grammar nit:

>A device lock pattern, PIN, or biometrics are required to protect your Autofill Login details

I feel like that "are" is wrong. Should it be "is" instead? i.e., 

A device lock pattern, PIN, or biometrics **is** required to protect your Autofill Login details

Reply:
Thanks for pointing this out! Confrimed here https://app.asana.com/0/0/1202418883857449/f and now updated.